warp10/src/main/java/io/warp10/script/processing/typography/PcreateFont.java

```diff
@@ -45,21 +82,68 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
 
     PFont font = null;
     
-    if (3 == params.size()) {
-      font = pg.parent.createFont(params.get(1).toString(), ((Number) params.get(2)).floatValue());
-    } else if (4 == params.size()) {
-      font = pg.parent.createFont(
-          params.get(1).toString(),
-          ((Number) params.get(2)).floatValue(),
-          Boolean.TRUE.equals(params.get(3)));
-    } else if (5 == params.size()) {
-      font = pg.parent.createFont(
-          params.get(1).toString(),
-          ((Number) params.get(2)).floatValue(),
-          Boolean.TRUE.equals(params.get(3)),
-          params.get(4).toString().toCharArray());
-    }
+    //
+    // Handle the case of remote font files differently, passing the URL
+    // to the defined validator
+    //
+    
+    if (params.get(1).toString().startsWith("http://") || params.get(1).toString().startsWith("https://")) {
```

Comment:
Could be nice to also allow `file://`.
(Tested and working with the SO solution below).